TWLight/emails/templates/emails/project_page_2021_email-body-text.html

```diff
@@ -0,0 +1,17 @@
+{% comment %} English-only, do not translate.{% endcomment %}
```

Comment:
Is this template used anywhere? I see that you don't import it in the `TWLight/emails/templates/emails/project_page_2021_email-body-html.html` file. Can we remove it?

Reply:
This gets loaded via [djmail's templatemail](https://bameda.github.io/djmail/#_templatemail)